Configuration/HLT/python/autoHLT.py

```diff
@@ -3,9 +3,9 @@
 
 autoHLT = {
   'frozen25ns' : '25ns14e33_v1',
-  'relval25ns' : '25ns14e33_v1',
+  'relval25ns' : '25ns14e33_v3',
   'frozen50ns' : '50ns_5e33_v1',
-  'relval50ns' : '50ns_5e33_v1',
+  'relval50ns' : '50ns_5e33_v3',
   'fake'       : 'Fake',
```

Comment:
(only for the relvals, not for production, and maybe only for 25ns) I think that moving to a more recent HLT menu would be a good idea - the usual problem is that we need to use a compatible L1 menu, and thus a different global tag...